pkgs/applications/editors/quartus-prime/default.nix

```diff
@@ -80,7 +80,11 @@ in buildFHSUserEnv rec {
     ln --symbolic --relative --target-directory ./bin $EXECUTABLES
   '';
 
+  # LD_PRELOAD fixes issues that cause memory corruption and crashes in certain containerized environments
```

Comment:
Can we be a bit more specific about what environments cause crashes? So that anybody updating this package can test whether the issue still exists in later versions.

From the forum post, it looks like WSL2?

Reply:
It's pretty much any container (I experienced it on LXC in Proxmox), and allegedly due to the licensing system having a buggy codepath handling the virtual network interfaces. The symptom is obvious: any of the `quartus_*` tools will die almost immediately complaining about memory corruption when they try to acquire a license to do real work (i.e. they will display the help fine, but not much more). If a basic design can be synthesized, then everything is working.